app/assets/javascripts/discourse/app/templates/modal/login.hbs

```diff
@@ -1,85 +1,83 @@
 {{#login-modal loginName=loginName loginPassword=loginPassword secondFactorToken=secondFactorToken action=(action "login")}}
   {{plugin-outlet name="login-before-modal-body" tagName=""}}
-  {{#d-modal-body title="login.title" class=modalBodyClasses}}
+  {{#d-modal-body class=modalBodyClasses}}
 
     {{#if canLoginLocal}}
-      <form id="login-form" method="post">
-        <div id="credentials" class={{credentialsClass}}>
-          <table>
-            <tbody>
-              <tr>
-                <td><label for="login-account-name">{{i18n "login.username"}}</label></td>
-                <td>{{text-field value=loginName placeholderKey="login.email_placeholder" id="login-account-name" autocorrect="off" autocapitalize="off" autofocus="autofocus" disabled=disableLoginFields}}</td>
-              </tr>
-              <tr>
-                <td><label for="login-account-password">{{i18n "login.password"}}</label></td>
-                <td>{{password-field value=loginPassword type="password" id="login-account-password" maxlength="200" capsLockOn=capsLockOn disabled=disableLoginFields}}</td>
-                <td><a href id="forgot-password-link" {{action "forgotPassword"}}>{{i18n "forgot_password.action"}}</a></td>
-              </tr>
-              <tr>
-                <td></td>
-                <td><div class="caps-lock-warning {{unless capsLockOn "hidden"}}">{{d-icon "exclamation-triangle"}} {{i18n "login.caps_lock_warning"}}</div></td>
-                <td></td>
-              </tr>
-            </tbody>
-          </table>
+      <div class="login-left-side">
+        <div class="login-welcome-header">
+          <h1>Welcome Back</h1> <img src="/images/emoji/twitter/wave/{{skinToneId}}.png" alt="" class="waving-hand">
+          <p>Sign in to your account</p>
```

Comment:
Translations are needed for both strings.
The image URL won't work with subfolder and the `emoji_set` site setting be used instead of hardcoding `twitter`?